daisy_tools/dmi/util.py

```diff
@@ -0,0 +1,40 @@
+from math import cos, asin, sqrt, pi
+from datetime import datetime
+
+__all__ = [
+    'construct_datetime_argument',
+    'distance',
+]
+
+def construct_datetime_argument(
+    from_time = None, to_time = None
+) -> str:
+    if from_time is None and to_time is None:
+        return None
+    if from_time is not None and to_time is None:
+        return f"{from_time.isoformat()}Z"
+    if from_time is None and to_time is not None:
+        return f"{to_time.isoformat()}Z"
+    return f"{from_time.isoformat()}Z/{to_time.isoformat()}Z"
```

Comment:
Lidt renere og tydeligere:

```suggestion
def construct_datetime_argument(
    from_time: datetime = None,
    to_time: datetime = None
) -> str | None:
    if from_time and to_time:
        return f"{from_time.isoformat()}Z/{to_time.isoformat()}Z"
    if from_time:
        return f"{from_time.isoformat()}Z"
    if to_time:
        return f"{to_time.isoformat()}Z"
    return None
```